### PR TITLE
LPS-164951 Create Output select to Formula field

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-164951

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/formulaFieldUtil.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+};
+
+export const FORMULA_OUTPUT_OPTIONS: FormulaOutput[] = [
+	{
+		description: Liferay.Language.get('select-between-two-possible-values'),
+		label: Liferay.Language.get('boolean'),
+		value: 'boolean',
+	},
+	{
+		description: 'for-date-values',
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/formulaFieldUtil.ts:28]: Hi @danvb112. I think here should be:

```suggestion
		description: Liferay.Language.get('for-date-values'),
```

Comment (danvb112) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/formulaFieldUtil.ts]: Thank you for seeing this @alinedoleron! Done! :smile:

**Comment by danvb112** [comment]
ci:test:sf

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**